data_validation/cli_tools.py

```diff
@@ -590,8 +590,12 @@ def _configure_row_parser(
     required_arguments.add_argument(
         "--primary-keys",
         "-pk",
-        required=True,
-        help="Comma separated list of primary key columns 'col_a,col_b'",
+        required=is_custom_query,
+        help=(
+            "Comma separated list of primary key columns 'col_a,col_b'" ""
```

Comment:
```suggestion
            "Comma separated list of primary key columns 'col_a,col_b'"
```
I think there's this additional string was added unintentionally

Reply:
Please add `default=None` before new line #593.